metricbeat/module/kubernetes/_meta/test/docs/README.md

```diff
@@ -1,88 +1,103 @@
-# Testing Metricbeat
+## Testing Metricbeat
 
+### Prerequisites
 
-## Create kubernetes cluster using kind
-
-Follow instructions at https://kind.sigs.k8s.io/docs/user/quick-start/#installation and install kind.
-
-Create a kind kubernetes cluster.
-```
-kind create cluster  --image 'kindest/node:v1.21.1'
-```
-
-## Deploy Kube-state-metrics
-
-Prerequisite for collecting kubernetes meaningful metrics is kube-state-metrics.
-Deploy it to your cluster manually by
+Make sure that the fields are update. If you don't know how, go to metricbeat directory and run:
 ```bash
-git clone git@github.com:kubernetes/kube-state-metrics.git
-cd kube-state-metrics/
+make update
+ ```
 
-kubectl apply -k .
-```
-
-## Create ELK stack
-
-You can spin up an ELK stack in two ways
-1. [Proposed] Using elastic cloud https://cloud.elastic.co
-2. Locally on your kind cluster (EK tuple will suffice).
+Check that the modules pass the tests. Go to the directory of each of them and run:
 ```bash
-# Deploy Elasticsearch and Kibana
-cd metricbeat/module/kubernetes/_meta/test/docs
-kubectl apply -f 01_playground/ek_stack.yaml
-
-# Expose Kibana with port forwarding. In your browser visit localhost:5601
-kubectl port-forward deployment/kibana 5601:5601
-```
-
-
-## Playground Metricbeat Pod
+go test -data
+ ```
+
+
+### Deploy metricbeat
```

Comment:
couple of notes about this doc:
- it is not very clear if `Deploy metricbeat` is required for the `go test` or the integration tests or the cloud deployments. We could add a few words making sure that it is clear that those are 4 different way to test metricbeat and they are independent from each other. 1. local go test, 2. manual testing with `Deploy metricbeat`, 3. integration tests, 4. tests in cloud
- given that we have 9 steps only for `deploy metricbeat`, it would be great to have a taskfile to provide a single command to the user
-